src/kyc_api.ts

```diff
@@ -0,0 +1,43 @@
+import axios, { AxiosInstance } from 'axios'
+
+export interface AccessToken {
+    token: string
+    userId: string
+}
+
+const BASE_URL = 'http://localhost:4000/api/v1'
+// const BASE_URL = 'https://kyc-poc-be.camino.foundation/api/v1'
```

Comment:
Question here, what should be the address of kyc backend? I think it's on the same server as the wallet app, so we should not have cors issues. Leaving it like that until I know for sure what to use here!

Reply:
Even if CORS errors arise, just let me know and ill add the required headers in nginx. i think sth like, `api.prod.camino.foundation/kyc/v1` will be the final url